docs/BreakingChanges/023 New (ambiguous) Dispatcher.Invoke overloads.md

```diff
@@ -0,0 +1,25 @@
+## 23: New (ambiguous) Dispatcher.Invoke overloads
+
+### Scope
+Minor
+
+### Version Introduced
+4.5
+
+### Change Description
+The .NET Framework 4.5 adds new overloads to Dispatcher.Invoke that include a parameter of type System.Action. When existing code is recompiled, compilers may resolve calls to Dispatcher.Invoke methods that have a Delegate parameter as calls to Dispatcher.Invoke methods with an System.Action parameter. If a call to a Dispatcher.Invoke overload with a Delegate parameter is resolved as a call to a Dispatcher.Invoke overload with an System.Action parameter, the following differences in behavior may occur:<ul><li>If an exception occurs, the Dispatcher.UnhandledExceptionFilter and Dispatcher.UnhandledException events are not raised. Instead, exceptions are handled by the UnobservedTaskException event.</li><li>Calls to some members, such as DispatcherOperation.Result, block until the operation has completed.</li></ul>
```

Comment:
This is still plain HTML syntax (from way back before these were markdown). We should probably change them to use markdown syntax so that the raw file is more readable.